src/pages/tBTC/Bridge/Mint.tsx

```diff
@@ -83,6 +87,11 @@ MintingFormPage.route = {
 
 const MintPageLayout: PageComponent = () => {
   const { active } = useWeb3React()
+  const { mintingStep, utxo } = useTbtcState()
+  const shouldRenderDurationWidget = ![
+    MintingStep.ProvideData,
+    MintingStep.Deposit,
+  ].includes(mintingStep)
```

Comment:
Let's also add undefined here to not display the `MintDurationWidget` at the loading step:

![image](https://github.com/threshold-network/token-dashboard/assets/40306834/1b433e8b-a74a-4079-b345-6b5b5cdb1d74)


```suggestion
  const shouldRenderDurationWidget = ![
    MintingStep.ProvideData,
    MintingStep.Deposit,
    undefined,
  ].includes(mintingStep)
```

Reply:
Ref commit: 3b075624